src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlDataReader.cs

```diff
@@ -162,7 +162,7 @@ override public int Depth
             {
                 if (this.IsClosed)
                 {
-                    throw ADP.DataReaderClosed("Depth");
```

Comment:
In this and other calls to DataReaderClosed why have you removed the parameter?

Reply:
`ADP.DataReaderClosed` has the `CallerMemberName` attribute attached to its parameter, so specifying a value isn't necessary in most cases (with exceptions for TryNextResult, TryReadInternal and TryReadColumn.)

It's one of the superficial changes which allow a cleaner diff between netfx and netcore - without these changes, SqlDataReader's diff is -1067/+630, with many of the changed areas also surrounding actual functionality changes. With the changes, that diff is -624/+165. The post-PR diff is also easier to read.